templates/base.html

```diff
@@ -210,15 +196,9 @@ <h4>Our Social Networks</h4>
 
     <!-- Template Main JS File -->
     <script src="{% static 'js/main.js' %}"></script>
-    <script>
```

Comment:
You have also removed this AOS script. Please carefully check all the files.

Reply:
@joetho786

Reply:
ok will do it asap